### DISCUSSION TITLE
Embed version number in release properties

### DISCUSSION THREAD

**Discussion by AbeKline** [rustdesk/rustdesk#7592] [General]
Could the build pipelines be updated to automatically include the version number in the file metadata? I'm specifically looking for it to be included in the RustDesk windows client executable. 
This would be very useful for managing deployments internally. We want to rehost it locally so we can control what version our clients have installed
As it stands, the Properties window has little to no information
![image](https://github.com/rustdesk/rustdesk/assets/8125653/01343dea-e37e-4fbc-a1c9-239799d57322)
Thanks!

**Comment by fufesou** [comment]
The nightly build has fixed this.

https://github.com/rustdesk/rustdesk/releases/tag/nightly

**Reply by AbeKline** [reply]
0.0
That was so fast. Thanks!

**Reply by aklineJC** [reply]
Please let me know if I should open an Issue for this instead of commenting here
On win10 there is a mismatch between the Registry Key version and the .exe installer version
`(Get-ItemProperty  "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\RustDesk\").Version`
yields
1.2.3.2
`(Get-ChildItem .\rustdesk-1.2.3-2-x86_64.exe).VersionInfo.[ProductVersion/FileVersion]`
yields 
1.2.3+39

Is there something I'm missing or is this expected behavior?
